### DISCUSSION TITLE
When to use `echo`?

### DISCUSSION THREAD

**Discussion by vwkd** [nushell/nushell#11758] [Q&A]
The help for the `echo` command states

> There is usually little reason to use this over just writing the values as-is.

When is using `echo` necessary?

**Comment by fdncred** [comment]
The way `echo` is supposed to work now is that if it's being redirected, it prints out a nushell value, the same as just writing the value itself. If it's just printing a "dumb" string, it work like print and does not print out a nushell value. You can work with values but you can't do much with strings.
